### PR TITLE
Control narrative updates for 3-year ATO

### PR DESCRIPTION
**Pull request by jseppi**
Work-in-progress during the ATO Sprint.

### CODE REVIEW

File: compliance/component.yaml [hunk 1]
```diff
@@ -112,30 +123,22 @@ satisfies:
...
-        hourglass/settings.py. Configurations which are
-        specific to one cloud.gov environment (i.e. either the dev, staging, or
-        production environment) are located in the appropriate manifest-*.yml
-        file or stored in and provided by a cloud.gov "user provided
-        service".
+        environments is located in the manifests directory of the code
+        repository. Each environment (dev, staging, production) has a separate
+        manifest file, which  is named "manifest-<environment label>.yml".
+        Application-level settings are specified in the code repository at
+        hourglass/settings.py.
```
Comment (jseppi) [inline compliance/component.yaml]: Should I say anything about user-provided services containing additional configuration values?

File: compliance/component.yaml [hunk 2]
```diff
@@ -25,15 +25,18 @@ satisfies:
...
-        of the site requiring login.
+        provided by the Django application. Only users who successfully login
+        through the UAA and who have an associated (via @gsa.gov email address)
+        account in the CALC back end will be able to access privileged areas of
+        the application.
+        Data Administrator users can access the applications' administrative
+        back end, where they can administer, including creating new, user
+        accounts for both Data Administrators and Contract Officers.
+        Contract Officer users are not able to access to access the
+        administrative back end or perform any actions on user accounts.
```
Comment (hollyallen) [inline compliance/component.yaml]: This phrasing is a bit awkward. What are the different privileged parts of the application, and which users can access which parts?

File: compliance/component.yaml [hunk 3]
```diff
@@ -88,20 +95,24 @@ satisfies:
 - standard_key: NIST-800-53
```
Comment (hollyallen) [inline compliance/component.yaml]: CA-8: Noah suggested specifying GSA Information Security team _or_ 3PAO but not both.

File: compliance/component.yaml (hunk 1 above)
Comment (hollyallen) [inline compliance/component.yaml]: Sounds like a good idea. In addition, a direct link to an example file would help the testers.

File: compliance/component.yaml (hunk 3 above)
Comment (NoahKunin) [inline compliance/component.yaml]: Just for the record, audio issue, other way around, let's keep the OR. :)

Comment (jseppi) [inline compliance/component.yaml]: Noah just corrected: keep the "and"

**Pull request merged**